axelor-crm/src/main/resources/views/Opportunity.xml

```diff
@@ -261,6 +265,14 @@
     	<field name="description" expr="eval: _partner.description"/>
     </action-record>
     
+    <action-attrs name="action-opportunity-attrs-description-panel-show">
+    	<attribute name="hidden" for="customerDescriptionPanel" expr="eval:!(__config__.app.getApp('crm').displayCustomerDescriptionInOpportunity)"/>
+    </action-attrs>
+    
+    <action-attrs name="action-opportunity-attrs-from-customer">
```

Comment:
don't add onChange action for customer but change action when opportunity is created from customer